config_builder.go

```diff
@@ -1471,8 +1471,40 @@ func initNeutrinoBackend(ctx context.Context, cfg *Config, chainDir string,
 			lncfg.SqliteNeutrinoDBName, lncfg.NSNeutrinoDB,
 		)
 
+	case cfg.DB.Backend == kvdb.PostgresBackendName:
+		// This code needs to be in place because we did not start
+		// the postgres backend for neutrino at the beginning. Now we
+		// are also moving it into the postgres backend so we can phase
+		// out the bolt backend.
+		dbName := filepath.Join(dbPath, lncfg.NeutrinoDBName)
+		// Check if the file exists, because we cannot start a new
+		// neutrino db before remove the old data associated with it.
+		exists := fileExists(dbName)
+		if exists {
+			ltndLog.Infof("Neutrino Bolt DB found, removing it " +
+				"and starting fresh with postgres")
+
+			// Remove the block_headers.bin file
```

Comment:
nit: missing full stop in comment. Also, function calls below should be formatted differently (since they don't use formatting directives).

Reply:
you mean with the err directive ?

Reply:
I mean this:
```
				blockHeadersPath := filepath.Join(
					dbPath, "block_headers.bin")
				regFilterHeadersPath := filepath.Join(
					dbPath, "reg_filter_headers.bin")
```

Should be:
```
				blockHeadersPath := filepath.Join(
					dbPath, "block_headers.bin",
				)
				regFilterHeadersPath := filepath.Join(
					dbPath, "reg_filter_headers.bin",
				)
```
to satisfy my code OCD :sweat_smile: